prow/github/report/report.go

```diff
@@ -250,13 +287,11 @@ func parseIssueComments(pj prowapi.ProwJob, botName string, ics []github.IssueCo
 			newEntries = append(newEntries, entries[i])
```

Comment:
Part of the intent here should be that if:
* jobs A B C reported for commit X,
* I push commit Y,
* when job B finishes for commit Y, create a table with just B's failure, not A and C (since these results are for the wrong commit).

Reply:
yes, i think we are already doing this.